grails-app/domain/mama/ng/scheduler/Message.groovy

```diff
@@ -0,0 +1,16 @@
+package mama.ng.scheduler
+
+class Message {
+
+    static hasOne = [schedule: Schedule]
+
+    String id = UUID.randomUUID().toString()
+    Date nextSend
+    Date dateCreated
+
+    static constraints = {
+        id generator:'assigned'
+        schedule(nullable: false)
+        nextSend(min: new Date(), nullable: false)
```

Comment:
Should we also require ``dateCreated`` to be non-null? Not quite sure of the value of requiring ``nextSend`` to be later than ``now`` -- seems slightly odd to have a constraint that will be violated at some point in the future.